embedded-graphics/src/style/text_style.rs

```diff
@@ -35,19 +42,100 @@ where
             background_color: None,
         }
     }
+
+    /// Create a text style with only the font defined
+    pub fn with_font(font: F) -> Self {
+        Self {
+            font,
+            text_color: None,
+            background_color: None,
+        }
+    }
+}
+
+/// Primitive style builder.
```

Comment:
This isn't the primitive style builder and it could use some additional docs (which the `PrimitiveStyleBuilder` could also use)

Reply:
Added some docs for both. Let me know what you think. Committed whilst tired af so might have missed something important...